user/pages/05.blog/2018-01-09-docker-improvements/item.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Running OpenNMS Horizon in Docker
+date: 21:48 2018-01-09
+author: Ronny Trommer
+body_classes: header-lite fullwidth blogstyling
+taxonomy:
+    category: docker
+    tag: [docker]
+---
+
+Running applications in containers provide a lot benefits and it's not just a hype.
+Higher velocity to maintain changes while keeping a service available, scaling software and teams up to abstracting your infrastructure to use your expensive hardware more efficiently - all this is the fuzz about running applications in containers.
```

Comment:
s/fuzz/buzz/
Unclear what you mean by "scaling software and teams up to abstracting your infrastructure".